resources/views/layouts/partials/footer/navigation.blade.php

```diff
@@ -0,0 +1,83 @@
+<nav class="grid grid-cols-2 gap-8 xl:col-span-2">
+    <div class="md:grid md:grid-cols-2 md:gap-8">
+        <div>
+            <h3 class="text-base font-medium text-gray-900">@lang('Catalog')</h3>
+            <ul role="list" class="mt-4 space-y-4">
+                <li>
+                    <a href="/what-is-new.html" class="text-base text-gray-500 hover:text-gray-900">
```

Comment:
This is repeated multiple times below, wasn't there a nice Tailwind thing so styled can be applied from the parent to the children? Otherwise maybe a php loop to reduce duplication?

Reply:
I used a template from Tailwind UI. But I agree with you, so I will change this to a php loop.